pkg/kubelet/kubelet_pods.go

```diff
@@ -1368,7 +1368,7 @@ func getPhase(spec *v1.PodSpec, info []v1.ContainerStatus) v1.PodPhase {
 		case containerStatus.State.Running != nil:
 			pendingInitialization++
 		case containerStatus.State.Terminated != nil:
-			if containerStatus.State.Terminated.ExitCode != 0 {
+			if containerStatus.State.Terminated.ExitCode != 0 || containerStatus.State.Terminated.Reason == "OOMKilled" {
```

Comment:
I am not seeing where the equivalent logic for this is handled in the linked PR.

Could you help point to the equivalent change here? 
see: https://github.com/kubernetes/kubernetes/pull/115331